connectivity/FEATURE_BLE/include/ble/gap/Events.h

```diff
@@ -575,15 +575,47 @@ struct PeriodicAdvertisingSyncLoss {
  */
 struct ScanTimeoutEvent { };
 
+/**
+ * Event produced when advertising start.
+ *
+ * @see ble::Gap::EventHandler::onAdvertisingStart().
+ */
+struct AdvertisingStartEvent {
+#if !defined(DOXYGEN_ONLY)
```

Comment:
why hide it from doxygen?

Reply:
Same as before, these objects are meant to be created by the BLE layer.